### DISCUSSION TITLE
What kind of Java apps is this for?

### DISCUSSION THREAD

**Discussion by xarx00** [coobbi/Jre4android#6] [General]
Hello. I'm glad to see an attempt to make Java running on Android. However...

What kind of Java apps is Jre4android planned to support? 
I tried a .jar game (a Java 8 game, The Abbey of Crime Extensum), and it immediately failed, probably on missing libraries. See the attached screenshot.
![Screenshot_20240912-181603](https://github.com/user-attachments/assets/69bf02eb-5562-41af-ac55-5fe0ae9bf3c4)

If java4android is meant for execution of Java console applications only, maybe it would be better to mention that in the app description to prevent incorrect expectations.

**Comment by coobbi** [comment]
Thanks for your suggestion. 
Jre4Android can run jar files with main methods, including command mode and gui mode (implemented by java swing). It does not support programs with attached .so files. I will update it in the description file later. 
The screenshot shows that the error message cannot load jsound lib. jsound depends on independent .so files.
